app/Libraries/Storage/FteStorageWrapper.php

```diff
@@ -4,7 +4,7 @@
 
 class FteStorageWrapper extends StorageWrapper
 {
-    protected $basePath = "storage/smartcars/exercises";
+    protected $basePath = "smartcars/exercises";
```

Comment:
Why are we changing the path?

Reply:
Because, to me at least, it doesn't make sense to store in public/storage/storage/smartcars...

Reply:
I think that was done due to some of the symlinking in prod; where storage/storage corresponded to public/storage

Reply:
it should be that storage/public maps to public/storage. If its not this is a prod issue not standard practice, which we probably need to look at fixing